docs/conf.py

```diff
@@ -58,6 +58,11 @@ def setup(sphinx):
     sphinx.connect("autoapi-skip-member", autoapi_skip_members)
 
 
+# -- AG CUSTOM CODE BABY
+def setup(app):
```

Comment:
Type checker is yelling about this function name shadowing an imported function. Should rename this